src/Admin/BaseFieldDescription.php

```diff
@@ -432,7 +432,7 @@ private function callCachedGetter(object $object, string $fieldName, array $para
         }
 
         if ('call' === self::$fieldGetters[$getterKey]['method']) {
-            return $object->__call($fieldName, $parameters);
+            return $object->{$fieldName}(...$parameters);
```

Comment:
I don't know if we should promote __call or `{}`.
The `__call` change was made on master I think.

Reply:
Change on 3.x: 6fa2e473c185f1a8d80758dbe0a17299867a05a9

Reply:
Change on master: 54e9f496440005b14171ea695b9f3998d279025e

Reply:
Oh indeed ! weird.